src/remotecontrol.cpp

```diff
@@ -47,7 +47,7 @@ void RemoteControl::handle()
     uint result = results.value;
     _IR_Receive.resume();
 
-    debugV("Received IR Remote Code: 0x%08X, Decode: %08X\n", result, results.decode_type);
+    debugW("Received IR Remote Code: 0x%08X, Decode: %08X\n", result, results.decode_type);

```

Comment:
Is it appropriate to make every IR button press issue a Warning level debug message? I'd say Verbose feels more suitable.

Reply:
I maintain a "W" is too strong. A warning is something the viewer should mildly worry about. If "V" is too weak then I think "I" could be the compromise; that's still included in the default debug output if I'm not mistaken.